.github/scripts/set-project-version.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+if [ ! -f "mvnw" ]; then
+  echo "⚠️ Maven Wrapper not found. You must execute this script from the project root directory. Exiting..."
+  exit 1
+fi
+
+if [ -z "$1" ]; then
+  echo "⚠️ You must provide the new version as an argument. Exiting..."
+  exit 1
+fi
+NEW_VERSION=$1
+CURRENT_VERSION=$(./mvnw help:evaluate -N -q -Dexpression=project.version -DforceStdout |tail -n 1)
+
+echo "🔍 Current project version is $CURRENT_VERSION, updating to $NEW_VERSION"
+echo "🔄 Updating version in pom.xml files"
+
+./mvnw versions:set -q -DnewVersion=$NEW_VERSION
+
+echo "🔄 Updating version in package.json files"
+PACKAGE_JSON_FILES=(\
+  $(find . -name package.json) \
+  $(find . -name package-lock.json) \
+)
+
+for PACKAGE_JSON_FILE in ${PACKAGE_JSON_FILES[@]}; do
+  sed -i '' -e "s/\"version\": \"$CURRENT_VERSION\"/\"version\": \"$1\"/" $PACKAGE_JSON_FILE
+done
+
+echo "🔄 Updating version in pom.xml files that are not part of the reactor"
```

Comment:
**Review Note**: I know this is a hack. But not all modules are currently in the reactor, so they can't be matched by the versions-maven-plugin.
Should be fixed later.